### PR TITLE
add mrinufft convention

### PR DESCRIPTION
**Pull request by paquiteau**
Also fixes the examples for the gifs.

### CODE REVIEW

File: docs/mrinufft_convention.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+K-Space Trajectories
+====================
+
+The k-space sampling trajectories are generated in the :py:`mrinufft.trajectories` and then use in the differents backends.
+They are ``numpy`` arrays with the followings characteristics:
+
+- ``float32`` or ``float64`` precision (this will trigger the use of single or double precision in the computations using this trajectory). ``float32`` precision is recommended for computational efficiency.
+
+- They are row major (C Convention) with the folowing shape: ``(N_shot, N_samples,dimD)`` (where ). A "flatten" version of shape ``(N_shots * N_samples, dim)``
```
Comment (Lenoush) [inline docs/mrinufft_convention.rst:80]: where what ?   I think part of the sentence is missing

File: README.rst [hunk 2]
```diff
@@ -88,3 +88,22 @@ MRI-nufft is available on Pypi and can be installed with::
...
 You will also need to install at least one NUFFT computation backend. See the `Documentation <https://mind-inria.github.io/mri-nufft/getting_started.html#choosing-a-nufft-backend>`_ for more guidance.
+
+
+Benchmark
+---------
+
+A benchmark of NUFFT backend for MRI applications is available in https://github.com/mind-inria/mri-nufft-benchmark
+
+
+Who is using MRI-NUFFT
```
Comment (chaithyagr) [inline README.rst:99]:
```suggestion
Who is using MRI-NUFFT?
```

File: docs/mrinufft_convention.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            ...
+            ):
+
+            self.samples = proper_trajectory(samples)
+            self.shape = shape
+            self.n_coils = n_coils
+            self.n_batchs = n_batchs
+
+            self.compute_smaps(smaps) # setup the smaps
+            self.compute_density(density) # setup the density compensation
```
Comment (chaithyagr) [inline docs/mrinufft_convention.rst:37]:
```suggestion
            self.compute_density(density) # setup the density compensation
            self.compute_smaps(smaps) # setup the smaps
```

Always we get density first, then Smaps

**Review by chaithyagr — CHANGES_REQUESTED** [review]
A few changes, great work. YOu can merge after handeling the issues (even from @Lenoush )

**Review by Lenoush — APPROVED** [review]

**Pull request merged**